sdk/eventhub/Microsoft.Azure.EventHubs/tests/Infrastructure/TaskExtensions.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.EventHubs.Tests
```

Comment:
I think we have a copy of this. Can we add a source reference to the existing file?

Reply:
Or put it into a shared test utility library?

Reply:
When I last spoke with Alex on shared test infrastructure, we had no immediate plans for it.   Since Event Hubs and Service Bus are separate SDKs and intended to be isolated, I'm not comfortable linking them.    At some point, it would be nice to have an `Azure.Core.Testing` or similar where we can share things kinds of things.    For now, I'm going to eat the small amount of duplication.